snuba/subscriptions/data.py

```diff
@@ -129,6 +141,57 @@ class LegacySubscriptionData(SubscriptionData):
 
     conditions: Sequence[Condition]
     aggregations: Sequence[Aggregation]
+    organization: Optional[int] = None
+    limit: int = 1
+    offset: int = 0
+    granularity: int = 3600
+
+    @staticmethod
+    def has_crash_rate_alert_aggregation(aggregations: Sequence[Aggregation]) -> bool:
+        for aggregation in aggregations:
+            try:
+                if aggregation[2] and bool(
+                    CRASH_RATE_ALERT_PATTERN.match(aggregation[2])
+                ):
+                    return True
+            except IndexError:
+                continue
+        return False
```

Comment:
Doesn't that mean anyone who just uses an alias containing `crash_free_percentage` in any subscription query will match the sessions behavior? It doesn't seem to check the dataset so it could even be applying this logic to events/transactions. This seems a rather unreliable way to determine whether how to process a subscription since this is something that may even be user defined in the future. I don't think I quite get why we need to do this at all?

Reply:
I changed the alias to a more specific name `_crash_rate_alert_aggregate` and now I ensured I only call it from within the Sessions entity .. does that resolve it?

Reply:
It was more about using an alias at all, it is not a reliable method of detecting a subscription even if we made it a more obscure name, we should find another way.